cernopendata/modules/fixtures/data/data-policies-v1.0.0.json

```diff
@@ -24,7 +24,7 @@
       {
           "uri": "root://eospublic.cern.ch//eos/opendata/lhcb/documentation/LHCb-Data-Policy.pdf",
           "size": 254288,
-          "checksum": "5e48e8042c8edc2f828770b41e21562b5570d689"
+          "checksum": "md:5e48e8042c8edc2f828770b41e21562b5570d689"
```

Comment:
```md:``` prefix need to be set in the checksum for invenio not to through an error, as of https://github.com/inveniosoftware/invenio-files-rest/blob/b1a30ebae9021c153c1466f978726511edbafcaf/invenio_files_rest/storage/base.py#L117-L121

@tiborsimko FYI for future data

Reply:
Actually I used SHA1 checksum there, not MD5... this one is less safe.

Reply:
BTW `eos cp --checksum` command seems to use adler32 by default for checksumming...